src/functions/getATFsLambda.js

```diff
@@ -1,7 +1,12 @@
-const getATFList = require('../services/getATFListService')
+const ATFService = require('../services/ATFService')
+const ATFDAO = require('../models/ATFDAOmock')
+const path = require('path')
 
 const getATFs = async () => {
-  return getATFList()
+  const DAO = new ATFDAO(path.resolve(__dirname, '../../src/mocks/mock-atf.json'))
+  const service = new ATFService(DAO)
+
+  return service.getATFList()
```

Comment:
In the catch, please console.log the error

Reply:
fixt